airflow/dag_processing/manager.py

```diff
@@ -280,7 +274,16 @@ def end(self):
         self._parent_signal_conn.close()
 
 
-class DagFileProcessorManager(LoggingMixin):
+def _config_int_factory(section: str, key: str):
+    return functools.partial(conf.getint, section, key)
+
+
+def _config_bool_factory(section: str, key: str):
+    return functools.partial(conf.getboolean, section, key)
+
+
+@attrs.define
+class DagFileProcessorManager:
```

Comment:
```suggestion
class DagFileProcessorSupervisor:
```

and let's rename the file appropriately.